client/components/main/style.scss

```diff
@@ -28,7 +27,6 @@
 	}
 
 	@include breakpoint( "<660px" ) {
-		backface-visibility: hidden;
 		perspective: 1000;
```

Comment:
Let's also try removing perspective 👍 This rule doesn't make sense outside of 3D elements.